pkg/controllers/operationjob/operationjob_controller.go

```diff
@@ -160,19 +155,24 @@ func (r *ReconcileOperationJob) doReconcile(ctx context.Context, instance *appsv
 	getErr := r.getTargetsOpsStatus(ctx, actionHandler, selectedCandidates, enablePodOpsLifecycle, instance)
 	// calculate opsStatus of all candidates
 	instance.Status = r.calculateStatus(instance, candidates)
-	return controllerutils.AggregateErrors([]error{opsErr, getErr})
+	// update operationjob status
+	updateErr := r.updateStatus(ctx, instance)
```

Comment:
If getErr is not nil, we should not update the status of opj?

Reply:
we should always update status to show details in opj